### DISCUSSION TITLE
FNIRSI GC-01: Right/settings button activates by itself

### DISCUSSION THREAD

**Discussion by Klempnertommy** [Gissio/radpro#121] [General]
First of all, thank you for this great firmware! Since the last firmware versions, the GC-01 automatically jumps into the menu after a few seconds. Is the problem already known?

https://github.com/Gissio/radpro/assets/2979027/0d139ed5-3380-41a6-be94-a5a0b3126e7e

**Comment by gc01noob** [comment]
I have the GC01 with APM CPU and J321 tube and this doesn't happen with 2.0.1 or 2.0.2

maybe you have an issue with the right hand switch?
is it only when charging/connected to usb?
have you selected the correct tube?

**Comment by Klempnertommy** [comment]
it is also without USB. It's really like this, so whether the right button is triggered randomly. It doesn't happen when the click comes, it happens randomly. I had the hardware click converted, could that be the problem? Or rather an overreaching of the high voltage

https://github.com/Gissio/radpro/assets/2979027/509744d7-ecac-43bf-8308-cc9f430c46fe

**Reply by jonwhite227** [reply]
My GC-01 has been doing this during pre-release testing, and still does it up to the previous release. If you go into the Geiger tube settings and select HV profile and set it to energy saving, it should stop this behavior with no effect on performance. There is a reason for this due to electro-magnetic interference near the right hand button, but Gissio would be able to explain it much better than me.

**Reply by Klempnertommy** [reply]
it looks like you were right. With energy saving, these problems no longer occur. EMF problem was also my first guess

**Comment by gc01noob** [comment]
I find the Energy Saving setting makes it less sensitive, I get almost zero background, so I leave it on Default and it all works fine. Mine is 100% stock.
There was another discussion about the non-APM CPU driving the tube at a much higher voltage - this was solved physically via a resistor or by creating a HV profile to drop the voltage.

You might want to add the specs of your device. eg CPU and Tube type
and also what "I had the hardware click converted" means - soldering on the board with exposed wires may be the problem

**Reply by Klempnertommy** [reply]
CPU is CH32, I chose the right firmware
Tube ist J613, I chose this tube
I will remove the hardware conversion soon, it is no longer needed with this beautiful firmware ;)
Maybe the default settings are set a little too high?

**Reply by Klempnertommy** [reply]
The hardware patch is now removed, but the problem has not changed. With the standard HV profile I get this strange effect. There doesn't seem to be any resistance on my board. Maybe that's why different settings are needed
![IMG_20240708_114508](https://github.com/Gissio/radpro/assets/2979027/5692fc74-0760-49a9-b585-f287e74ccfcc)
![IMG_20240708_114603](https://github.com/Gissio/radpro/assets/2979027/dfd8fa43-ef07-46b2-be0d-f52a78bf0a40)

**Reply by jonwhite227** [reply]
I don't get any noticeable decrease in sensitivity, but I swapped out the J613/J614 tube for a J321, which actually made it show about 3x background. I was going to adjust the HV using resistors, but installing RadPro solved the problem for me. I get around 0.14uSv/h here.

**Reply by Klempnertommy** [reply]
arrrrg... 
Did I say I have the CH32 processor?
Did I just take a photo myself with an APM32?
Maybe I should choose the right firmware...

**Reply by Klempnertommy** [reply]
Problem solved!
I'm so sorry, the problem only way between my ears... 
thanks a lot, there is a great community here. 
BTW Flashing in Linux works, when you set the rights for write before and overwrite the file twice
